crates/wasmtime/src/runtime.rs

```diff
@@ -194,10 +195,16 @@ impl Config {
         self.validating_config
             .operator_config
             .enable_reference_types = enable;
-        // The reference types proposal depends on the bulk memory proposal
+
+        self.flags
+            .set("enable_safepoints", if enable { "true" } else { "false" })
```

Comment:
Out of curiosity, could we always enable this? Presumably once reference types ships we'll enable this unconditionally anyway.

Reply:
We could, but it implies an additional pass over the IR by cranelift, and if we know there aren't any being used, then we can skip that unnecessary work.

Reply:
This seems fine for now, but I wanted to point this out because that optimization is something we need to likely implement in short order in that case. In theory there should be a fast path for functions which don't use anyref?

Put another way, if this is scoped to only reference types for performance reasons, then we need to file an issue about that and get it fixed before fully shipping reference types.

Reply:
Filed https://github.com/bytecodealliance/wasmtime/issues/1883